lib/cc/engine/csslint.rb

```diff
@@ -52,6 +27,35 @@ def run
 
       private
 
+      def create_issue(node, path)
+        begin
```

Comment:
Now that the `rescue`d code is it's own method you don't need the explicit `begin`

```rb
def create_issue(node, path)
  # ...
rescue KeyError => ex
  # ...
end
```